docs/documentation.md

```diff
@@ -570,9 +570,9 @@ You can follow along [on my technical blog](https://ttboj.wordpress.com/).
 
 To report any bugs, please file a ticket at: [https://github.com/purpleidea/mgmt/issues](https://github.com/purpleidea/mgmt/issues).
 
-##Authors
+## Authors
 
-Copyright (C) 2013-2016+ James Shubin and the project contributors
+Copyright © 2013-2016+ James Shubin and the project contributors
```

Comment:
> Can you comment on why we need to make this change?

Well, it's a proper markdown syntax. Some markdown implementations will not convert `#Header` headers without whitespaces. CommonMark for example.
>I'd rather not make this (C) change, unless you have a good reason why.
I thought it should be proper copyright symbol too, since it's 2017 and we have unicode.